root_main.tf

```diff
@@ -34,4 +34,28 @@ module "ses-eu-west-1" {
 
 module "security_hub" {
   source = "./tdr-terraform-modules/securityhub"
-}
\ No newline at end of file
+}
+
+module "encryption_key" {
+  source      = "./tdr-terraform-modules/kms"
+  project     = var.project
+  environment = local.environment
+  common_tags = local.common_tags
+  function    = "account"
+}
+
+module "cloudtrail-s3" {
```

Comment:
Nitpick: this name uses hyphens, and the other modules use snake_case. Unless this is consistent with how we're doing all the cloudtrail modules?

Reply:
thanks, corrected